presto-hive/src/main/java/com/facebook/presto/hive/HiveWriteUtils.java

```diff
@@ -408,6 +415,47 @@ public static void createDirectory(HdfsEnvironment hdfsEnvironment, Path path)
         }
     }
 
+    public static boolean checkTypeWritable(HiveType hiveType)
```

Comment:
Why is this method called `checkTypeWritable` and not `isWritableType`? The later one is easier to understand - check suggested me that it might throw something.